script.js

```diff
@@ -0,0 +1,66 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const bookForm = document.getElementById('add-book');
+  const list = document.querySelector('.book-list');
+
+  let books = [];
+
+  function displayBooks(id, title, author) {
+    const items = document.createElement('li');
+    list.style.listStyleType = 'none';
+    items.innerHTML = `
+      <p>Title:-  ${title}</p>
+       <p> Author:-  ${author}</p>
+       `;
+    const removeBtn = document.createElement('button');
+    removeBtn.textContent = 'Remove';
+    removeBtn.id = id;
+    const bookLine = document.createElement('hr');
+    items.append(removeBtn, bookLine);
+
+    // REMOVE BOOKS
+    removeBtn.addEventListener('click', () => {
+      books = books.filter((book) => {
+        if (book.id === id) {
+          return false;
+        }
+        return true;
+      });
```

Comment:
- [ ] As per this [requirement,](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=title%20and%20author.-,Create%20a%20function%20to%20remove%20a%20book%20from%20the%20collection%20(hint%3A%20you%20can%20use%20the%20array%20filter()%20method).,-Display%20all%20books) Please create a separate function to remove a book from the books collection then you can call it in the event listener.